.github/workflows/vast.yaml

```diff
@@ -461,24 +496,36 @@ jobs:
             docker tag tenzir/${{ matrix.docker-vast.name }}:latest "ghcr.io/tenzir/${{ matrix.docker-vast.name }}:${ref_tag}"
             docker push "ghcr.io/tenzir/${{ matrix.docker-vast.name }}:${ref_tag}"
           fi
+      - name: Login to Docker Hub
+        if: github.event_name == 'push' || github.event_name == 'release'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Configure AWS Credentials
+        # if: contains(matrix.docker-vast.registries, '622024652768.dkr.ecr.eu-west-1.amazonaws.com')
+        if: matrix.docker-vast.name == 'vast-manager'
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: ${{ secrets.AWS_INFRASTRUCTURE_GITHUB_ACTIONS_ROLE }}
+          aws-region: ${{ secrets.AWS_INFRASTRUCTURE_DEFAULT_REGION }}
+      - name: Login to AWS ECR
+        # if: contains(matrix.docker-vast.registries, '622024652768.dkr.ecr.eu-west-1.amazonaws.com')
+        if: matrix.docker-vast.name == 'vast-manager'
+        uses: aws-actions/amazon-ecr-login@v1
       - name: Publish Docker Images
-        if: ${{ github.event_name == 'push' || github.event_name == 'release' }}
+        if: github.event_name == 'push' || github.event_name == 'release' || (matrix.docker-vast.name == 'vast-manager' && inputs.upload-docker-image-to-ecr)
```

Comment:
Make these conditionals more precise. Make the registry selection more fine-grained in the matrix.

Reply:
Will switch to a reusable workflow that supports this, but that has to land on master first it seems.